backend/app/services/abstract_enrollments_extractor.rb

```diff
@@ -15,7 +15,9 @@ def call
   end
 
   def query_enrollments
-    Enrollment.where(status: extract_criteria[:statuses])
+    Enrollment
+      .where(status: extract_criteria[:statuses])
```

Comment:
En terme de lisibilité, serait il possible de passer le `where.not` ligne 19 ?

Reply:
yes